src/openrct2/ParkFile.cpp

```diff
@@ -971,9 +971,9 @@ namespace OpenRCT2
                     cs.Write(static_cast<uint32_t>(numBanners));
 
                     [[maybe_unused]] size_t numWritten = 0;
-                    for (BannerIndex i = 0; i < MAX_BANNERS; i++)
+                    for (uint16_t i = 0; i < MAX_BANNERS; i++)
```

Comment:
I guess there is no way to do a `for` loop of actual `BannerIndex`es?

Reply:
It could be implemented but I don't think one should do arithmetic with those handles, I could probably expose the underlying type so when the type changes no code needs updating.